Product/Production/Services/PatientCorrelationCore/src/main/java/gov/hhs/fha/nhinc/patientcorrelation/nhinc/proxy/PatientCorrelationProxyWebServiceUnsecuredImpl.java

```diff
@@ -179,4 +185,31 @@ public AddPatientCorrelationResponseType addPatientCorrelation(final PRPAIN20130
         LOG.debug("End addPatientCorrelation");
         return response;
     }
+    @Override
+    public void addPatientCorrelationPLQ(PatientLocationQueryResponseType plqRecords,
+        AssertionType assertion) {
+        AddPatientCorrelationPLQRequestType request = new AddPatientCorrelationPLQRequestType();
+        request.setPatientLocationQueryResponse(plqRecords);
+        request.setAssertion(assertion);
+
+        try {
+            final String url = oProxyHelper.getUrlLocalHomeCommunity(NhincConstants.PATIENT_CORRELATION_SERVICE_NAME);
+            if (plqRecords == null) {
+                LOG.error(NULL_ERROR);
+            } else if (assertion == null) {
+                LOG.error(NULL_ERR_ASSERTION);
+            } else {
+
+                final ServicePortDescriptor<PatientCorrelationPortType> portDescriptor = new PatientCorrelationAddPLQServicePortDescriptor();
+
+                final CONNECTClient<PatientCorrelationPortType> client = getCONNECTClientFactory()
+                    .getCONNECTClientUnsecured(portDescriptor, url, assertion);
+
+                client.invokePort(PatientCorrelationPortType.class,"addPatientCorrelationPLQ", request);
+            }
+        } catch (final Exception ex) {
```

Comment:
Dont catch generic exception

Reply:
Remaining consistent with the rest of the application.